src/components/RenderArticles/RenderArticles.jsx

```diff
@@ -1,22 +1,37 @@
 import { useState, useEffect } from 'react'
-import { getArticles } from '../../utils/utils'
+import apiCall from '../../hooks/apiCall'
 import { ArticleItem } from './components/ArticleItem/ArticleItem'
 import { LoadMoreBtn } from '../LoadMoreBtn/LoadMoreBtn'
 import './RenderArticles.css'
 
-export const RenderArticles = () => {
+export const RenderArticles = ({queries}) => {
```

Comment:
rather than having a whole new component to handle this - is there a way we could reuse our original articles component/utils and amend it so it can take topic queries into account?